client/src/components/icons/DeleteIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import {TouchableOpacity, View} from "react-native";
+import {MaterialCommunityIcons} from "@expo/vector-icons";
+import {IconsStyles} from "./FavoriteIcon";
+import React from "react";
+
+interface Props {
+    itemId: string;
```

Comment:
itemId is not used. Will be better when refactoring all icons as mentioned

Reply:
done